tests/kafkatest/services/kafka/kafka.py

```diff
@@ -323,8 +326,12 @@ def start_cmd(self, node):
         cmd += "export KAFKA_LOG4J_OPTS=\"-Dlog4j.configuration=file:%s\"; " % self.LOG4J_CONFIG
         heap_kafka_opts = "-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=%s" % \
                           self.logs["kafka_heap_dump_file"]["path"]
+        # https://docs.confluent.io/current/kafka/deployment.html#jvm
+        prod_kafka_opts = ("-Xms{0} -Xmx{0} -XX:MetaspaceSize=96m -XX:+UseG1GC -XX:MaxGCPauseMillis=20 ".format(self.jvm_heap_size)
```

Comment:
Instead of using `.format` and `+` to create the string, maybe use same way as `cmd` is constructed (using % to format, and multiline string without `+` but by ending with `\`)

Reply:
I used `.format` to avoid having to pass the same arg twice since it seems you can't use the `"{0} {0}" % (arg)` in python2 

The `\` vs `+` makes sense